src/osp/UserInputHandler.cpp

```diff
@@ -245,8 +245,21 @@ void UserInputHandler::config_register_control(std::string const& name,
         std::initializer_list<ButtonVarConfig> vars)
```

Comment:
As I said in the .h file, there's no need to have a version of this function that takes an initializer list. It always gets converted to std::vector, so just keep that version(s) of the function.